multi_sensor:
  ros__parameters:
    calibrate: true # calibrate function must be improved
    # Gyro range: 0 -> +-250°/s, 1 -> +-500°/s, 2 -> +-1000°/s, 3 -> +-2000°/s
    gyro_range: 1
    # Acce range: 0 -> +-2g, 1 -> +-4g, 2 -> +-8g, 3 -> +-16g
    accel_range: 1
    # DLPF bandwidth [0-6]: 0 -> 260Hz, 1 -> 184Hz, 2 -> 94Hz, 3 -> 44Hz, 4 -> 21Hz, 5 -> 10Hz, 6 -> 5Hz
    dlpf_bandwidth: 3
    #If "calibrate" is true, these values will be overriden by the calibration procedure
    accel_x_offset:  0.08     #  0.0 # [m/s²]
    accel_y_offset: -0.32     #  0.0 # [m/s²]
    accel_z_offset: -0.24     #  0.0 # [m/s²]
    gyro_x_offset:   2.68     #  0.0 # [deg/s]
    gyro_y_offset:  -0.69     #  0.0 # [deg/s]
    gyro_z_offset:   0.46     #  0.0 # [deg/s]
    frequency: 520 # [Hz]